android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -0,0 +1,43 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.ui.theme.Blue
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun FavoriteButton(
+    modifier: Modifier = Modifier,
+    contentDescription: String = "Favourite button"
+) {
+    val favourite = remember { mutableStateOf(false) }
```

Comment:
Favorite state should come from "the outside". Meaning as a function parameter. This is in general a good practice in compose to keep the components as dumb as possible and to cache the state somewhere else. 
You can read more about it here: https://developer.android.com/jetpack/compose/mental-model
Protip: `isFavorite` is a better name than `favorite` since it communicates what the data actually represents more clearly :)